vocabularies/voc4cat/0007017.ttl

```diff
@@ -0,0 +1,17 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007017>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007017"^^xsd:token ;
+    skos:definition "A process equipment where chemicals are fed in order to make them chemically react with each other."@en ;
```

Comment:
You could make the def. it a bit more general. This def. excludes batch reactors (no feed).

Reply:
are the chemicals "placed" in the reactor to achieve a conversion?

Reply:
I have changed the definition to `A process equipment to which elevated temperature and pressure can be applied to convert chemicals into desired products, often in the presence of a catalyst.`

Reply:
Or should I add the possibility of a light source too, so that is also includes photoreactors? @nmoust @dalito

Reply:
There are many reactor types. Should we define here a concept `chemical reactor` (to distinguish from not a nuclear reactor) instead of `reactor`?

For `chemical reactor`, I suggest: `An apparatus or system designed for the controlled execution of chemical reactions.`

For `reactor` I suggest `An apparatus or system designed for the controlled execution of reactions, encompassing a wide range of processes including chemical, nuclear, biochemical, or other physical transformations.`

An interesting question is: What does the concept "reactor" include except the vessel? Are for example containment vessels, control mechanisms, safety devices etc. part of reactor? - No need to answer this here!

Reply:
Good points, I include definitions for `reactor`, `chemical reactor` and `catalytic reactor`. Instead of `apparatus or system` I suggest the more general `enclosed volume`.

Reply:
We can use the concept "reactor setup" which can include all the surrounding equipment (e.g, pressure gauges, MFCs, etc.) and reactor having the definition that @dalito mentioned before.

Reply:
ok great! I included the concept "reactor setup".